spectroscopy/code_src/sdss_functions.py

```diff
@@ -32,40 +24,40 @@ def SDSS_get_spec(sample_table, search_radius_arcsec, data_release):
     -------
     df_lc : MultiIndexDFObject
         The main data structure to store all spectra
-        
+
     '''
 
-    
-    ## Initialize multi-index object:
+    # Initialize multi-index object:
     df_spec = MultiIndexDFObject()
-    
+
     for stab in sample_table:
 
-        ## Get Spectra for SDSS
+        # Get Spectra for SDSS
         search_coords = stab["coord"]
-        
-        xid = SDSS.query_region(search_coords, radius=search_radius_arcsec * u.arcsec, spectro=True, data_release=data_release)
+
+        xid = SDSS.query_region(search_coords, radius=search_radius_arcsec *
+                                u.arcsec, spectro=True, data_release=data_release)
 
         if str(type(xid)) != "<class 'NoneType'>":
-            sp = SDSS.get_spectra(matches=xid, show_progress=True , data_release=data_release)
-    
-            ## Get data
-            wave = 10**sp[0]["COADD"].data.loglam * u.angstrom # only one entry because we only search for one xid at a time. Could change that?
-            flux = sp[0]["COADD"].data.flux*1e-17 * u.erg/u.second/u.centimeter**2/u.angstrom 
+            sp = SDSS.get_spectra(matches=xid, show_progress=True, data_release=data_release)
+
+            # Get data
+            # only one entry because we only search for one xid at a time. Could change that?
+            wave = 10**sp[0]["COADD"].data.loglam * u.angstrom
+            flux = sp[0]["COADD"].data.flux*1e-17 * u.erg/u.second/u.centimeter**2/u.angstrom
             err = np.sqrt(1/sp[0]["COADD"].data.ivar)*1e-17 * flux.unit
-            
-            ## Add to df_spec.
-            dfsingle = pd.DataFrame(dict(wave=[wave] , flux=[flux], err=[err],
-                                     label=[stab["label"]],
-                                     objectid=[stab["objectid"]],
-                                     mission=["SDSS"],
-                                     instrument=["SDSS"],
-                                     filter=["optical"],
-                                    )).set_index(["objectid", "label", "filter", "mission"])
+
+            # Add to df_spec.
+            dfsingle = pd.DataFrame(dict(wave=[wave], flux=[flux], err=[err],
+                                         label=[stab["label"]],
+                                         objectid=[stab["objectid"]],
+                                         mission=["SDSS"],
+                                         instrument=["SDSS"],
+                                         filter=["optical"],
+                                         )).set_index(["objectid", "label", "filter", "mission"])
             df_spec.append(dfsingle)
 
         else:
             print("Source {} could not be found".format(stab["label"]))
 
-
-    return(df_spec)
\ No newline at end of file
+    return (df_spec)
```

Comment:
not sure if we can clean all these up without using ruff or something similar. Just leaving the comment for the example
```suggestion
    return df_spec
```

Reply:
I'll take them out when fix the docstrings since I'll be checking the return objects then. This is much easier to do manually than, say, deciding where to break long lines of code.